lib/std/os.zig

```diff
@@ -5012,12 +5012,12 @@ pub fn sigaltstack(ss: ?*stack_t, old_ss: ?*stack_t) SigaltstackError!void {
 }
 
 /// Examine and change a signal action.
-pub fn sigaction(sig: u6, act: ?*const Sigaction, oact: ?*Sigaction) void {
+pub fn sigaction(sig: u6, noalias act: ?*const Sigaction, noalias oact: ?*Sigaction) error{OperationNotSupported}!void {
     switch (errno(system.sigaction(sig, act, oact))) {
         .SUCCESS => return,
-        .FAULT => unreachable,
-        .INVAL => unreachable,
-        else => unreachable,
+        // .FAULT => unreachable,
```

Comment:
Dead code is rather uncool, so you should make a comment, why the code stays for now.